lib/Cargo.toml

```diff
@@ -15,7 +15,7 @@ secp256k1 = ["ssi/libsecp256k1", "did-tezos/secp256k1", "did-key/secp256k1"]
 #didkit_cbindings = { path = "cbindings/" }
 ssi = { path = "../../ssi", default-features = false }
 did-key = { path = "../../ssi/did-key" }
-did-tezos = { path = "../../ssi/did-tezos" }
+did-tezos = { path = "../../ssi/did-tezos" , optional = true}
```

Comment:
`lib/src/did_methods.rs` should be updated to check for the `did-tezos` feature now.

https://github.com/spruceid/didkit/runs/2023683074#step:5:533

Reply:
Also we might need `default-features = false` since `did-tezos` now has [`default = ["ssi/ring"]`](https://github.com/spruceid/ssi/commit/bda41612ba31a6cc4b4ed1765f6fddac978652b5#diff-cc014949a5cfda49e81ef7744cb81f64326bbc492501b11f0ed601069016c6faR8)

Reply:
`did-tezos` can then be enabled for CLI:
https://github.com/spruceid/didkit/runs/2023900714#step:9:13
```diff
diff --git a/cli/Cargo.toml b/cli/Cargo.toml
index a3fb6f0..f29d335 100644
--- a/cli/Cargo.toml
+++ b/cli/Cargo.toml
@@ -13,3 +13,3 @@ chrono = { version = "0.4", features = ["serde"] }
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
-didkit = { path = "../lib", features = ["did-web", "http-did"] }
+didkit = { path = "../lib", features = ["did-web", "http-did", "did-tezos"] }
 serde = { version = "1.0", features = ["derive"] }
```
For HTTP:
```diff
diff --git a/http/Cargo.toml b/http/Cargo.toml
index d1be443..80c2fb7 100644
--- a/http/Cargo.toml
+++ b/http/Cargo.toml
@@ -11,3 +11,3 @@ ring = ["ssi/ring"]
 [dependencies]
-didkit = { path = "../lib", features = ["did-web", "http-did"] }
+didkit = { path = "../lib", features = ["did-web", "http-did", "did-tezos"] }
 didkit-cli = { path = "../cli" }
```

Probably will need `optional = true` for `did-tezos` in `lib/wasm/Cargo.toml`:

```diff
diff --git a/lib/wasm/Cargo.toml b/lib/wasm/Cargo.toml
index a625406..2696e54 100644
--- a/lib/wasm/Cargo.toml
+++ b/lib/wasm/Cargo.toml
@@ -29,8 +29,9 @@ features = ["secp256k1"]
 
 [dependencies.did-tezos]
 path = "../../../ssi/did-tezos"
 features = ["secp256k1"]
+optional = true
 
 [dev-dependencies]
 wasm-bindgen-test = "0.2"
 
```